membership-common/src/main/scala/com/gu/memsub/subsv2/Plan.scala

```diff
@@ -431,12 +422,4 @@ case class CatalogPlan[+P <: Product, +C <: ChargeList, +S <: Status](
     saving: Option[Int],
     charges: C,
     s: S,
-) extends Plan[P, C]
-
-/** So the benefit of all these type parameters on the higher level models is that you can uniquely identify a particular plan by its type signature
-  * and if you can do that then you can pass your super specific (or more generic) plan into subscription service to find the subscription of your
-  * dreams
-  */
-object SubscriptionPlan {
-  type AnyPlan = SubscriptionPlan[Product, ChargeList]
```

Comment:
a subscription has a subscriptionplan by default, all the type parameters are gone